### ISSUE TITLE
Unable to export season to csv

### ISSUE THREAD

**Issue by wendybo** [BurntSushi/nflgame#357]
3 months ago, someone changed the code in seq.py from a set to a dictionary. It appears to break exporting an entire season to a csv file. The error I got was in line 306 'dict' object has no attribute 'add'. Changing that object back to a set seemed to succeed in terms of being able to export the data to a csv file.

**Comment by nehashah264** [comment]
Same error, @BurntSushi can you check and let us know?

**Comment by BurntSushi** [comment]
@nehashah264 No. Please see the README: https://github.com/BurntSushi/nflgame/commit/c9747ccb597d6c31f61e4002bc4bd9f45a05b064

**Comment by wendybo** [comment]
@nehashah264 I have a fork with the fix, find it here https://github.com/wendybo/nflgame

**Comment by rs327008** [comment]
@wendybo thanks that worked for me
